evalai/challenges.py

```diff
@@ -67,10 +69,57 @@ def list_future_challenges():
     get_future_challenge_list()
 
 
+@click.group(invoke_without_command=True)
+@click.pass_context
+@click.option('-c',
```

Comment:
Set `type=int` here and for `-p` so you get them as integers.
Also, add a long option
```
@click.option('-c', '--challenge-id', type=int,
```